tests/unit/_pip_sync_test.py

```diff
@@ -61,7 +61,11 @@ def test_it_writes_the_requirements_out_to_be_compiled(
         RequirementList.from_strings.assert_called_once_with(
             Any.generator.containing(["-creqs.txt", ".[tests]"]).only()
         )
-        requirements_set.constrained_set.assert_called_once_with()
+        requirements_set.constrained_set.assert_called_once_with(
+            # This path is the difference between the tox dir and the project
+            # root
```

Comment:
```suggestion
            # This path is the difference between the venv dir and the project
            # root
```

The path from the `.tox` dir to the project root would be `../`. I think you must mean from the `.tox/<ENV>/` dir?

Reply:
Yep!